omod/src/main/webapp/pages/visit/templates/encounters/hivNotMigratedEncounterLong.gsp

```diff
@@ -0,0 +1,30 @@
+<div class="header" ng-include="'templates/encounters/defaultEncounterHeader.page'">
+</div>
+
+<div class="content encounter-summary-long">  <!-- encounter-summary-long currently only used for Selenium tests -->
+    <p class="bold">
+        ${ ui.message("pihcore.hiv.legacyDataNotMigrated") }
+    </p>
```

Comment:
This is basically just a copy of defaultEncounterLong template with the above message added... we can tweak as needed as others review.